echidna/output/plot_chi_squared.py

```diff
@@ -86,49 +86,41 @@ def chi_squared_vs_signal(signal_config, converter=None, fig_num=1,
     # Set the tick labels, via Axes instance
     ax = fig.gca()  # Get current Axes instance
     for label in (ax.get_xticklabels() + ax.get_yticklabels()):
-        label.set_fontsize(MAIN_FONT.get("size"))  # Set other properties here e.g. colour, rotation
-
-    if kwargs.get("save_as") is not None:
-        plt.savefig(kwargs.get("save_as") + ".png", dpi=400)
+        # Set other properties here e.g. colour, rotation
+        label.set_fontsize(MAIN_FONT.get("size"))
 
-    if kwargs.get("show") is not None:
+    if save_as:
+        plt.savefig(save_as + ".png", dpi=400)
+    if show:
         plt.show()
     return fig
 
 
-def chi_squared_map(syst_analyser, fig_num=1, **kwargs):
+def chi_squared_map(syst_analyser, fig_num=1, preferred_values=True,
+                    minima=True, contours=False, save_as=None):
     """ Plot chi squared surface for systematic vs. signal counts
 
     Args:
-      syst_analyser (:class:`echidna.limit.limit_setting.SystAnalyser`): Systematic
-        analyser object, created during limit setting. Can be used
-        during limit setting setting or can load an instance from
-        hdf5
+      syst_analyser (:class:`echidna.limit.limit_setting.SystAnalyser`):
```

Comment:
sphinx-build warning:
```
/home/ashley/snoplus/software/echidna/echidna-pr/echidna/output/plot_chi_squared.py:docstring of echidna.output.plot_chi_squared.chi_squared_map:6: WARNING: Block quote ends without a blank line; unexpected unindented.
```
I think you have to have a lease one word after the colon otherwise the formatting isn't correct.

![screenshot from 2015-09-07 16 42 25](https://cloud.githubusercontent.com/assets/1931666/9719785/e42390ba-557f-11e5-8b45-c5f24f6028b1.png)